video/transmux.go

```diff
@@ -197,7 +197,9 @@ func concatStreams(segmentList, outputTsFileName string) error {
 		"f":    "concat", // Use stream based concatenation (instead of file based concatenation)
 		"safe": "0"}).    // Must be 0 since relative paths to segments are used in segmentListTxtFileName
 		Output(outputTsFileName, ffmpeg.KwArgs{
-			"c": "copy", // Don't accidentally transcode
+			"c:v": "copy", // Don't accidentally transcode video
+			"c:a": "aac",  // Re-encode audio to avoid audio/video sync issues
+			"af":  "aresample=async=100",
```

Comment:
Could we have a comment on this line too please?

Reply:
Updated